map_name: "point_cloud.ply"
separate:
  enable: true
  size: 200.0
  min_size: 10
  save: true
  ground_name: "ground.pcd"
  nonground_name: "nonground.pcd"
  resolution: 0.1
  max_diff: 1.0
  max_height: 1.5
  cluster_resolution: 0.005
  cluster_distance: 0.1
  near_min: 0.5
  near_max: 1.0
  erase_fudge: 1.0
  min_distance_considered: 4.0
  highest_diff: 1.0

image:
  intensity: false
  height: false
  resolution: 0.1
  max_height: 0.3
